feat: adding Chrissy Codes

CBID2 opened the pull request:
## Issue 
This PR adds my name to the list

CBID2 committed:
feat: adding Chrissy Codes

CBID2 committed:
feat: adding bid image

CBID2 committed:
feat: adding more information

CBID2 commented:
Hi @XanderRubio! :) Can you help me figure out what went wrong with the deployment?

XanderRubio commented:
> Hi @XanderRubio! :) Can you help me figure out what went wrong with the deployment?

Hi @CBID2,

Thank you for taking the time to contribute and reiewing now😎

XanderRubio committed:
Update CONTRIBUTORS.md

Updated the format to prevent a merge conflict later on during the merge of this pull request to main branch

XanderRubio commented on CONTRIBUTORS.md:
I made a format edit after updating the current list from the main branch to avoid a merge conflict on my end and added your GitHub avatar link so that your image is displayed on the CONTRIBUTORS.md file.

XanderRubio committed:
Update Contributors.json

Updated a syntax error with the JSON and added contributors city location based off LinkedIn profile .

XanderRubio commented on src/Jsons/Contributors.json:
I made some changes to the JSON file to fix a syntax error. Specifically, I added a comma that was missing. Additionally, I included your city location and state based on the information from your LinkedIn profile. The reason for doing this is that we are planning to create a Google Maps API (or other React Map Library) component that will display the locations of contributors. We want to make sure that this component has a smooth flight or scroll effect when it is displayed in the text box of contributors' "Before I die" statements. This will be helpful when we fetch the data from the contributors and build out this component.

XanderRubio reviewed (approved):
Great news! Your contribution is good to go @CBID2! I appreciate you sharing your desire to master Javascript. It's a valuable skill that will constantly evolve, and there will always be room for growth and mastery. Even with the help of AI tools, the potential to learn and the opportunities that Javascript creates for software development are vast and global and will always need that human touch. Thank you for sharing your enthusiasm with us!

XanderRubio commented:
Congratulations @CBID2 for your pull request being successfully merged<img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png" alt="Bottle with Popping Cork" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png" alt="Bottle with Popping Cork" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Food/Bottle%20with%20Popping%20Cork.png" alt="Bottle with Popping Cork" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Red%20Exclamation%20Mark.png" alt="Red Exclamation Mark" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Red%20Exclamation%20Mark.png" alt="Red Exclamation Mark" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Symbols/Red%20Exclamation%20Mark.png" alt="Red Exclamation Mark" width="25" height="25" />

Your contribution looks great on the [live link](https://before-i-die-achievements.vercel.app/)!
If you haven't already, please go ahead and star <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png" alt="Glowing Star" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png" alt="Glowing Star" width="25" height="25" /><img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Travel%20and%20places/Glowing%20Star.png" alt="Glowing Star" width="25" height="25" />this repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Chrissy!

Xander
